src/client/manager.ts

```diff
@@ -196,14 +190,6 @@ export class ClientManager {
         return this.connectionErrors;
     }
 
-    /**
-     * Get all allowed tools
-     * @returns Set of tool names that are allowed without confirmation
-     */
-    getAllowedTools(): Set<string> {
-        return this.confirmationProvider.allowedTools;
-    }
-
```

Comment:
removing this because this just routes to a dependency

this will only be needed in the CLI/UI.

We will move all initialization to the root of the project in the CLI/UI/wherever and expose APIs/functions as needed on top - best practices for dependency injection. improves testability, configurability and readability